common/config/azure-pipelines/integration-validation.yaml

```diff
@@ -23,35 +23,38 @@ jobs:
       matrix:
         Linux_node_18_x:
           imageName: ubuntu-latest
+          poolName: # Leaving blank defaults to image. Excluding field will cause errors in yml
```

Comment:
adding this to every empty field is ugly. Does only adding it to first field seem sufficient?

Reply:
yeah, the first occurrence of imageName and poolName should be enough